releases/release-1.27/release-notes/maps/pr-113121-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 113121
+releasenote:
+  text: cacheSize field in `EncryptionConfiguration` is not supported for KMSv2 provider
```

Comment:
Can you please backtick the name `cacheSize`. Also need to `|-` so that we can backtick the first word